pgjdbc/src/main/java/org/postgresql/core/v3/ConnectionFactoryImpl.java

```diff
@@ -228,7 +228,7 @@ public QueryExecutor openConnectionImpl(HostSpec[] hostSpecs, String user, Strin
         return queryExecutor;
       } catch (UnsupportedProtocolException upe) {
         // Swallow this and return null so ConnectionFactory tries the next protocol.
-        LOGGER.log(Level.SEVERE, "Protocol not supported, abandoning connection.", upe);
+        LOGGER.log(Level.FINE, "Protocol not supported, abandoning connection.", upe);
```

Comment:
Frankly speaking, I would like to kill `catch (UnsupportedProtocolException upe)` altogether (and `UnsupportedProtocolException` as well)  since the support of v2 protocol is not there.